test/utils.js

```diff
@@ -1,3 +1,5 @@
+"use strict";
```

Comment:
Every other file has this in. It was suggested adding this into all files might fix the npm install problem. It didn't - but lets get this in every file anyway.